src/variables/utils/astim.ts

```diff
@@ -19,11 +19,19 @@ const parseAllVariables = (ast: File): MemberExpression[] => {
 }
 
 export const parseASTIM = (query: string): ASTIM => {
-  const ast: File = parse(query)
-  // Using `any` to circumvent ts error
```

Comment:
Removed use of `any`  type. Which indicated that the code was not handling 2 possible children of MemberExpression.

We now handle both cases, both as if else. (Some future proofing in case there is a possible AST node child of `variable.property` which we don't know about in the UI types.